### PR TITLE
Subscription checkbox frontend (covid-19)

### PR DESCRIPTION
**Pull request by CodeBubble**
Added a subscription checkbox in checkout for covid-19 meals page
Passing down `isSubscription` and `causeHandle` props to be formatted into the subscriptionInfo object in `StripePaymentForm.jsx`

### CODE REVIEW

File: packages/frontend/components/SubscriptionPage/Checkout.jsx [hunk 1]
```diff
@@ -217,7 +217,7 @@ class Checkout extends React.PureComponent {
       amount: totalAmount.toFixed(2),
       serviceFee: serviceTip,
       subscriptionInfo: {
-        causeHandle: 'la',
+        causeHandle: 'greece',
```
Comment (lucashu1) [inline packages/frontend/components/SubscriptionPage/Checkout.jsx:220]: *refugee (to be consistent with the DB)

File: packages/frontend/components/StripeForm/StripePaymentForm.jsx [hunk 2]
```diff
@@ -53,201 +53,157 @@ class StripePaymentForm extends Component {
...
+  isUserInputValid = paymentObject => {
     const {
       amount,
-      serviceFee,
       stripeInfo: { donorInfo },
+      honoreeInfo,
     } = paymentObject;
 
-    const { firstName, lastName, email } = donorInfo || {};
+    console.log('userinput valid', paymentObject);
```
Comment (lucashu1) [inline packages/frontend/components/StripeForm/StripePaymentForm.jsx:63]: Remove this before merging

**Comment by Raghav-Maheshwari** [comment]
@lucashu1 @CodeBubble So I think this PR is ready to be reviewed again. It:

- Adds to `/stripeWebhook` to handle future invoice payments for meals
- Refactors `async submit()` in `stripePaymentForm` to handle both one time charges and subscriptions
- Adds a monthly subscription toggle to the meals checkout page:
image
- Updates success page for subscriptions:
image

File: packages/frontend/components/StripeForm/StripePaymentForm.jsx [hunk 3]
```diff
@@ -53,201 +53,155 @@ class StripePaymentForm extends Component {
...
+      captureObj = {
+        ...paymentObject,
+        stripeInfo: {
+          ...paymentObject.stripeInfo,
+          stripeToken: token.id,
+        },
+      };
+    }
+
+    console.log('captureObj', captureObj);
```
Comment (lucashu1) [inline packages/frontend/components/StripeForm/StripePaymentForm.jsx:202]: Remove remaining console.logs

File: packages/frontend/components/StripeForm/StripePaymentForm.jsx (hunk 2 above)
Comment (lucashu1) [reply packages/frontend/components/StripeForm/StripePaymentForm.jsx:63]: bump

File: packages/frontend/pages/covid-19/success/index.jsx [hunk 4]
```diff
@@ -206,9 +234,38 @@ class Covid19SuccessPage extends React.PureComponent {
...
+                    classes.subtitleMobile
+                  )}
+                >
+                  <div>Monthly Donation #{donation.donationId}</div>
+                  <div>
+                    Next donation:&nbsp;
+                    {this.getNextSubscription()}
+                  </div>
+                </div>
+              </React.Fragment>
```
Comment (lucashu1) [inline packages/frontend/pages/covid-19/success/index.jsx:263]: Might be cleaner to separate this (as well as the single-donation counterpart) into its own `const` variable? But not strictly necessary

**Pull request merged**